torch/testing/_internal/common_methods_invocations.py

```diff
@@ -8825,8 +8825,10 @@ def __init__(
         arity: int = 3,
         rightmost_supports_scalar: bool = False,
         rightmost_supports_scalarlist: bool = False,
+        without_values_kwarg: bool = False,
```

Comment:
What's the whole deal with the values kwarg??

Reply:
I'm fixing it in https://github.com/pytorch/pytorch/pull/112781 and reverted this change for this PR.